src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.nav {
+  /* border: 5px solid pink; */
+  display: flex;
+  height: auto;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  /* height: 60px; */
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.nav__list {
+  list-style: none;
+  /* border: 5px solid blue; */
+  display: flex;
+  flex-grow: 2;
+  justify-content: flex-end;
+
+  margin: auto;
+}
+
+a {
+  display: flex;
+  /* align-items: center; */
+  margin: 10px;
+  /* padding: 10px; */
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  /* border: 5px solid yellow; */
+  border-bottom: 4px solid transparent;
+}
+
+a:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+a:active {
```

Comment:
use .is-active::after and create line with position absolute, also add position relative for .is-active